src/main/java/nl/tudelft/dnainator/parser/buffered/DefaultEdgeParser.java

```diff
@@ -86,22 +107,30 @@ private int eatSpaces(int next) throws IOException {
 	private String parseDest() throws IOException, InvalidEdgeFormatException {
 		StringBuilder dest = new StringBuilder(ID_LENGTH_GUESS);
 		int point = eatSpaces(br.read());
+		boolean destParsed = false;
 		char next;
 
-		while (point != -1) {
+		parseLoop: while (point != -1) {
```

Comment:
https://gist.github.com/gfokkema/3459d91a610253e3f4f9

(I hate labels, especially when they're not necessary :p.